File: ledger-service/src/main/java/zachkingcade/dev/ledger/adapter/in/web/dto/journal/JournalLineDTOEnrichedResponse.java
package zachkingcade.dev.ledger.adapter.in.web.dto.journal;

public record JournalLineDTOEnrichedResponse(
        Long id,
        Long amount,
        Long accountId,
        String accountName,
        String accountTypeName,
        String accountDisplayName,
        char lineAffectOnAccount,
        char direction,
        String notes
) {
}

File: user-interface-service/src/app/adapter/ledger-service/dto/Journal-entry/JournalLineDTOEnrichedResponse.ts
export interface JournalLineDTOEnrichedResponse {
  id: number;
  amount: number;
  accountId: number;
  accountName: string;
  accountTypeName: string;
  accountDisplayName: string;
  lineAffectOnAccount: string;
  direction: string;
  notes: string;
}
